vault/core_cache_invalidate_test.go

```diff
@@ -169,7 +172,7 @@ func TestCore_Invalidate_Namespaces(t *testing.T) {
 	}, 10*time.Second, 10*time.Millisecond)
 }
 
-func TestCore_Invalidate_Namespaces_NonTransactional(t *testing.T) {
+/*func TestCore_Invalidate_Namespaces_NonTransactional(t *testing.T) {
```

Comment:
```suggestion
func TestCore_Invalidate_Namespaces_NonTransactional(t *testing.T) {
```

Just ran this test locally, it works.